RocketChat/Rocket.Chat
# fix: Message mentions displayed in the UI are outdated after changing username

## Proposed changes (including videos or screenshots)

Reparse the markdown (`md`) field when updating messages that mention the username changed.




## Issue(s)


## Steps to test or reproduce

- Send a message mentioning a user.
- Change the user's username
- Check if the mentioned message is still working
(test also the `DISABLE_MESSAGE_PARSER` env config to see if it is correctly disabling the parsing)

## Further comments

SUP-378
